aws-java-workshop/src/main/java/com/emc/vipr/s3/sample/_08_CreateLargeObject.java

```diff
@@ -20,10 +20,12 @@
 import java.io.BufferedReader;
 import java.io.File;
 import java.io.InputStreamReader;
-import java.util.ArrayList;
-import java.util.List;
+import java.lang.reflect.Array;
+import java.util.*;
+import java.util.concurrent.*;
 
 public class _08_CreateLargeObject {
+	protected final AmazonS3 client = AWSS3Factory.getS3Client();
```

Comment:
It's not another instance...It's the same instance being retrieved for this particular function.
there are a couple of different variations of demonstrating how to create large object either via single thread or multithreaded...The reference could be passed into the method, but there's not need because it's a public static object so the function can just grab it when it needs it